Bio/SeqUtils/__init__.py

```diff
@@ -399,7 +400,7 @@ def six_frame_translations(seq, genetic_code=1):
 
     >>> from Bio.SeqUtils import six_frame_translations
     >>> print(six_frame_translations("AUGGCCAUUGUAAUGGGCCGCUGA"))
-    GC_Frame: a:5 t:0 g:8 c:5 
+    GC_Frame: a:5 t:0 g:8 c:5
```

Comment:
You (or more likely your editor) has accidentally removed a trailing space here, which broke the doctest.

In this particular line, perhaps the function doesn't need the trailing space... but that's a separate issue.

Reply:
Thanks for catching that!